internal/httpapi/httpapi.go

```diff
@@ -123,10 +130,50 @@ func (s *Server) Serve(ctx context.Context) {
 		}()
 	}
 
+	c := http.Client{}
+	var ingestReadyOrDisabled, statusReadyOrDisabled bool
+	for {
+		if !ingestReadyOrDisabled && s.cfg.EnableIngest {
+			ingestReadyOrDisabled = s.isGetSuccessful(c, fmt.Sprintf("http://%s:%d%s", s.cfg.HostIngest, s.cfg.PortIngest, ingestAPIPathReady))
```

Comment:
does `ingestReadyOrDisabled` contain the information about being disabled? It looks to me that it should be just  `ingestReady` (same for status below)

Reply:
you are right! fixed